staff/daniel-toledo/gamma-template/about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>About-Gamma</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header__menu">
+        <h2 class="header__logo">Gamma</h2>
+            <i class="fas fa-bars header__burguer"></i>
+            <nav class='header__nav'>
+                <a href="home.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="contact.html">Contact</a>
+            </nav>
+        </div>
+
+        <div class="header__page-title">
+            <h1>About Gamma</h1>
+        </div>
+
+    </header>
+
+    <main class='about'>
+        <div class='about__info'>
+            <h2 class="content__title">intrigueproof</h2>
+            <p class='content__text'>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quo sequi expedita harum quae veritatis perspiciatis maiores cum iusto, porro et est. Nobis in illum repudiandae, consequatur dolore asperiores fugit quo! </p>
+            
+            <div class='about__collap'>
```

Comment:
You could try to find a more semnatical tag instead of div

Reply:
changed to `<section>`